### DISCUSSION TITLE
Setting options from command line

### DISCUSSION THREAD

**Discussion by doquang** [idaholab/moose#19149] [Q&A General] [answered]
I would like to turn off the color output when running an app. The command given by -h saying: "**--no-color** _Disable coloring of all Console outputs_." does not work. For example,
./moose_test-opt --no-color -i tests/kernels/simple_diffusion/simple_diffusion.i has no effect.

I only success by setting it via the input file
```
[Outputs]
  color = false
[]
```

**Comment by GiudGiud** [comment, marked as answer]
Hello

When you used no color you must have received the following warning:
```

*** Warning, This code is deprecated and will be removed in future versions:
The --no-color flag is deprecated. Use '--color off' instead.
```
--color off worked for me

Guillaume

**Comment by doquang** [comment]
Oh, sorry, It showed up on the very first lines that I missed.
